src/routes/StackView/StackView.scss

```diff
@@ -13,8 +15,61 @@ $stack-view-top-margin: 100px;
 
 .stack-view {
   margin-top: $stack-view-top-margin;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: fit-content;
 }
 
 .stack-view__disabled-click {
   pointer-events: none;
 }
+
+.stack-view__close-button {
+  all: unset;
+  position: absolute;
+  top: 0;
+  right: 0;
+  margin: $margin--default;
+  cursor: pointer;
+  width: 42px;
+  color: $color-white;
+  transition: all 0.08s ease-out;
+
+  &:hover {
+    transform: scale(1.3);
+  }
+
+  &:active {
+    transform: scale(1);
+  }
+
+  &:focus-visible {
+    color: var(--accent-color);
+  }
+}
+
+.stack-view__parent-note {
+  margin: 0;
+  width: calc(100vw - 36px);
```

Comment:
Please refer to the other comment about the calc subtraction and use a variable instead of a magic number if possible